apps/flame-defi/package.json

```diff
@@ -72,6 +73,9 @@
     "ts-node": "^10.9.2",
     "typescript": "5.5.4"
   },
+  "peerDependencies": {
+    "jsbi": "^3.2.5"
```

Comment:
This reminds me reason I first used JSBI was actually because I copied some logic from Uniswap types lol. Should we switch our plan to now refactor out big.js in favor of jsbi?

Reply:
sgtm!